src/fastmcp/server/auth/cimd.py

```
@@ -0,0 +1,634 @@
+            assertion: JWT assertion string from client
+            client: OAuth proxy client (must have cimd_document)
+            token_endpoint: Token endpoint URL for aud validation
+
+        Returns:
+            True if assertion is valid
+
+        Raises:
+            ValueError: If client doesn't have CIMD document or validation fails
+        """
+        if not hasattr(client, "cimd_document") or not client.cimd_document:
+            raise ValueError("Client must have CIMD document for private_key_jwt")
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)